### DISCUSSION TITLE
trivy scanner not working version 0.41.0 for docker images

### DISCUSSION THREAD

**Discussion by prateektrivy** [aquasecurity/trivy#7687] [Q&A]
### Question

Hi Team,

We are using trivy scanner for linux machines from past 1 year for scanning docker images, we have upgraded it recently as that version was obsolete with version 0.41.0, this version was working for 1 or 2 months then after that we have started getting error related to DB error and rate limit error.
error "DB: failed to download vulnerability DB: failed to list releases: GET https://api.github.com/repos/aquasecurity/trivy-db/releases: 403 API rate limit exceeded for 51.141.00.000. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) ".

we tried many solutions but it didn't worked out, we used GitHub token as the solution for authenticating it but still the issue persists.

Please help in resolving the issue.

Thanks
DevOps Team

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by simar7** [comment]
hi @prateektrivy please see https://github.com/aquasecurity/trivy/discussions/7699
